tamr_unify_client/models/attribute_configuration/collection.py

```diff
@@ -0,0 +1,77 @@
+from tamr_unify_client.models.attribute_configuration.resource import (
+    AttributeConfiguration,
+)
+from tamr_unify_client.models.base_collection import BaseCollection
+
+
+class AttributeConfigurationCollection(BaseCollection):
+    """Collection of :class'tamr_unify_client.models.attribute_configuration.resource.AttributeConfiguration'
```

Comment:
This needs to have backticks instead of quotes to render properly in the docs (:class\`~tamr_unify_client.models.attribute_configuration.resource.AttributeConfiguration\`)